src/views/OrderDetail.vue

```diff
@@ -402,6 +403,18 @@ export default defineComponent({
         })
         this.isParentProductUpdated = false;
       }
+    },
+    async configureDateTimeFormat(arrivalDate: string) {
+      const dateTimeConfigModal = await modalController.create({
+        component: DateTimeConfigModal,
```

Comment:
Improve name, we are not configuring the date time, but the format
```suggestion
        component: ConfigureDateTimeFormatModal,
```